### PR TITLE
[RPA-850] Hero section scaling fix

### PR DESCRIPTION
**Pull request by Aeri4a**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

**Review by Aeri4a — COMMENTED** [review]
self-rev

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/HeroSection/HeroBackground/HeroBackground.module.scss [hunk 1]
```diff
@@ -155,9 +155,28 @@ $ARROWS_HEIGHT: 176px;
...
 }
 
+
 @media (min-width: $breakpoint-xxl) {
     .heroArrows {
         background-position: center;
     }
 }
 
+@media (min-height: 700px) {
```
Comment (preludium) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/HeroSection/HeroBackground/HeroBackground.module.scss]: stick to defined breakpoints in breakpoint css file

**Review by preludium — DISMISSED** [review]

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/HeroSection/HeroBackground/HeroBackground.module.scss (hunk 1 above)
Comment (preludium) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/HeroSection/HeroBackground/HeroBackground.module.scss]: media queries are designed to work with the width property. Try to solve it the other way

Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/HeroSection/HeroBackground/HeroBackground.module.scss]: we're implementing desktop-first approach - try to use only max-width

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/HeroSection/HeroBackground/HeroBackground.module.scss [hunk 2]
```diff
@@ -8,25 +8,25 @@ $ARROWS_HEIGHT: 176px;
...
+    justify-content: space-between;
+    height: 730px;
 }
 
 .heroHandWrapper {
     position: absolute;
     z-index: -1;
     width: 100%;
-    height: 120vh;
+    height: 1000px;
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/HeroSection/HeroBackground/HeroBackground.module.scss]: if you have const value move it to the top of the file to const var e.g.:
HAND_WRAPPER_HEIGHT: 1000px;

**Review by star-in-the-jar — COMMENTED** [review]
bump version

**Review by star-in-the-jar — APPROVED** [review]

**Pull request merged**